yb-voyager/cmd/eventQueue.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/tgtdb"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
+)
+
+const (
+	QUEUE_DIR_NAME               = "queue"
+	QUEUE_SEGMENT_FILE_NAME      = "segment"
+	QUEUE_SEGMENT_FILE_EXTENSION = "ndjson"
+)
+
+type EventQueue struct {
+	QueueDirPath       string
+	SegmentNumToStream int64
+}
+
+func NewEventQueue(exportDir string) *EventQueue {
+	return &EventQueue{
+		QueueDirPath:       filepath.Join(exportDir, "data", QUEUE_DIR_NAME),
+		SegmentNumToStream: 0,
+	}
+}
+
+// GetNextSegment returns the next segment to process
+func (eq *EventQueue) GetNextSegment() (*EventQueueSegment, error) {
+	log.Infof("getting next segment to process from %s", eq.QueueDirPath)
```

Comment:
This line will flood the log file. No need to print/log anything in the `GetNextSegment()`.